Functions/Quantification/xASL_quant_FSL.m

```diff
@@ -106,16 +106,30 @@
 
     %% 4. Create option_file that contains options which are passed to the FSL command
     % FSLOptions is a character array containing CLI args for the BASIL/FABBER command
-	% Define if BASIL or FABBER is used - multiTE needs FABBER 
-	if (isfield(x.modules.asl, 'bQuantifyMultiTE') && x.modules.asl.bQuantifyMultiTE) || bQuantifyMultiTE
-		bUseFabber = 1;
-        FSLfunctionName = 'fabber_asl';
+	
+	% Define if BASIL or FABBER is used - multiTE needs FABBER. VABY is an alternative
+	% Allow external input of quantification type
+	if isfield(x, 'external') && isfield(x.external, 'ExternalQuantificationType')
```

Comment:
I don't see why we need to rename the string. If we need to specifically for VABY, we should do this in a subfunction.

We can rename this function to xASL_quant_External and have a VABY-specific subfunction, if VABY uses a lot of the settings similarly to FSL?

In any case, it would be good to split:
A. preparation of Tex/ATT/fitting parameters
B. specific parameters for the quantification modeling

Reply:
previously, we had only FSL and then we renamed BASIL or FABBER based on input data... Now this changes.

VABY is very very similar, so we should keep it in a single function. 
ok xASL_quant_External.

We are of course splitting based on specific parameter...

Reply:
Fine to call it xASL_quant_External. But still, it seems that the parameter choices, model choices, and software (Vaby/ExploreASL later/FSL now) are a bit mixed.

So 1: in the near future I would like to separate these.

And 2:
My problem is that we are changing 99% of the code usage (FSL) for 1% of the code usage (Vaby, we won't use that many times). That doesn't seem like good practice.

So instead of:

if Vaby
     .....
else 
   ..... (what we had originally, which you have to move complete sections of code)
end

I would prefer:

[keep what we have]

at the end have a separate section

if Vaby
    ......
end

So it is very clear that Vaby is a single time exception, that we probably will remove in the near future. Right?

You can of course insert this at multiple locations, but trying to keep the original code unchanged.

If you do it this way, I'm fine not changing the name, keeping everything as is (except perhaps the FSL folder name arranging or something) and just inserting Vaby here and there as a "quick and dirty feature" which we will soon remove when we have our own tooling for this?

Reply:
- [ ] Rename xASL_quant_FSL to xASL_quant_External
- [ ] Make the VABY/FABBER/FSL choices cleaner